tests/php/sync/test_class.jetpack-sync-themes.php

```diff
@@ -65,4 +65,83 @@ public function test_changed_theme_is_synced() {
 
 		$this->assertEquals( $local_value, $this->server_replica_storage->get_option( 'theme_mods_' . $this->theme ) );
 	}
+
+	public function test_widgets_changes_get_synced() {
```

Comment:
This isn't a blocking comment, but consider different tests for the different widget sync event types.